CVE-2022-22965/cvex.yml

```diff
@@ -0,0 +1,10 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+ubuntu2:
+  playbook: ubuntu2.yml
+  trace: "curl"
+  command:
+    - "python3 /opt/spring4shell/exploit.py --url \"http://%ubuntu1%:8080/helloworld/greeting\""
+    - "sleep 30"
```

Comment:
Is the sleep really necessary? Why?

Reply:
The vulnerable app takes a while to start. Can only curl after it starts